packages/ui/src/components/settings/RenderingSettings.tsx

```diff
@@ -69,6 +69,16 @@ export function RenderingSettings() {
           />
         </Group>
       )}
+      <Group>
+        <Label>{meta.rendering.groupByScene.name}</Label>
```

Comment:
i'm not sure if this meant to be pulled from the `meta` to be honest. `MetaField` documentation mentions something like this: https://github.com/motion-canvas/motion-canvas/blob/1c259d0d574bb56dbc8bc448300d9b94ee4d0bc4/packages/core/src/meta/MetaField.ts#L25-L31
while no other fields use it. depending on your answer i can either change it all to read from `MetaField` value, or use string literal here instead.

Reply:
alright, I'll just use string literal here then.